boards/xtensa/esp32s2/esp32s2-saola-1/src/esp32s2_bringup.c

```diff
@@ -154,6 +158,15 @@ int esp32s2_bringup(void)
 
 #endif /* CONFIG_TIMER */
 
+#ifdef CONFIG_ESP32S2_RT_TIMER
+  ret = esp32s2_rt_timer_init();
+  if (ret < 0)
+    {
+      syslog(LOG_ERR, "Failed to initialize RT timer: %d\n", ret);
+      return ret;
```

Comment:
```suggestion
```
Even if the RT Timer initialization fails, the bringup function should not return early.

Reply:
Done